docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -0,0 +1,135 @@
+= Seeding instances
+
+If you are a project architect you can configure the instances key parameters in the Controls window, before migration. The parameters can be added and edited as needed.
+The following parameters define applications, individuals, teams, verticals or areas within an organization affected or participating in the migration:
+
+* Stakeholders
+* Stakeholder groups
+* Job functions
+* Business services
+* Tag types
+* Tags
+
+You can create and configure an instance in any order. However, the suggested order below is the most efficient for creating stakeholders and tags.
+
+Stakeholders:
+
+. Create stakeholder groups
+. Create job functions
+. Create stakeholders
+
+Tags:
+
+. Create tag types
+. Create tags
+
+Stakeholders and defined by:
+
+* Email
+* Name
+* Job function
+* Stakeholder groups
+
+== Creating a new stakeholder group
+
+You can follow these steps to create a new stakeholder group. There are no default stakeholder groups defined.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Stakeholder groups*.
+. Click *Create new*.
+. Enter the following information:
+** Name
+** Description
+** Member(s)
+. Click *Create*.
+
+== Creating a new job function
+
+{ProductName} ({ProductShortName}) uses the job function attribute to classify stakeholders and provides a list of default values that can be expanded.
+
+You can follow these steps to create a new job function, which is not in the default list.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Job functions*.
+. Click *Create new*.
+. Enter a job function title into the *Name* text box.
+. Click *Create* button.
+
+== Creating a new stakeholder
+
+You can follow these steps to create a new migration project stakeholder.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Stakeholders*.
+. Click *Create new*.
+. Enter the following information:
+** Email
+** Name
+** Job function - custom functions can be created
+** Stakeholder group
+. Click *Create*.
+
+== Creating a new business service
+
+{ProductName} ({ProductShortName}) uses the business services attribute to define the departments within the organization that use the application and that are affected by the migration.
+
+[NOTE]
+====
+Business services must be created before creating or importing applications.
+====
+
+There are no default business services defined and they must be created before creating the applications.
+
+You can follow these steps to create a new business service.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Tags*.
+. Click *Create tag*.
+. Enter the following information:
+** Name
+** Rank - the order which the tags appear on the applications
+** Color
+. Click *Create* button.
+
+== Creating new tag types
+
+{ProductName} ({ProductShortName}) uses tags to classify applications in multiple categories. Follow the steps below to create a new tag type, which is not in the default list.
+
+.Procedure
+
```

Comment:
This is the procedure to create Tags, not Tag Types

Reply:
I took the information from here, https://github.com/konveyor/konveyor.github.io/blob/main/content/Tackle/instances.md

1. Should Creating New Tag Types come before Creating New Tags? 

2. Both "Creating New Tags" and "Creating New Tag Types" have a step called - Click "Create tag types". 
Does that step belong in both procedures?
What is the correct procedure for Creating Tags and Creating Tag Types?

Thanks!

Reply:
Corrected based on SCoons feedback.